app/models/abilities/everyone.rb

```diff
@@ -3,6 +3,10 @@ class Everyone
     include CanCan::Ability
 
     def initialize(user)
+      can :access, :ckeditor   # needed to access Ckeditor filebrowser
+      can [:access, :read, :create, :destroy], Ckeditor::Picture
```

Comment:
Note that with this configuration any user can delete any image, even the ones uploaded by different users.

Reply:
To consider: this model allows actions for **literally everyone**, even non-logged/registered users

The proper model to use is `common.rb`